python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```diff
@@ -84,14 +84,16 @@ keys:
                   type: dict
                   keys:
                     id:
-                      type: int
+                      type: str
                       convert_types:
-                        - str
-                      description: VLAN ID.
+                        - int
+                      description: |-
+                        VLAN ID or range of VLAN ID.
                     vni:
-                      type: int
+                      type: str
                       convert_types:
-                        - str
+                        - int
+                      description: VNI or range of VNI.
```

Comment:
since we are creating separate key for `vlan_range`, we should keep the `id` and `vni` as earlier i.e.`int`

Reply:
Fixed.